dashboard/app/jobs.go

```diff
@@ -666,6 +667,47 @@ func gatherCrashTitles(req *dashapi.JobDoneReq) []string {
 	return ret
 }
 
+// resetJobs is called by syz-ci to "unstart" previously started jobs.
```

Comment:
I'm reading this comment with bottom one and don't understand what is "unstart":
*//   - syz-ci sends JobResetReq to indicate that no previously started jobs
//     are any longer in progress.*
Could you please rephrase it?